addons/fleet/views/fleet_vehicle_cost_views.xml

```diff
@@ -41,17 +44,13 @@
                                 <span class="w-25">
                                     <field name="cost_generated" attrs="{'invisible': [('cost_frequency','=','no')]}" widget="monetary"/>
                                 </span>
-                                <field name="cost_frequency"/>
-                            </div>
```

Comment:
formatting in stable.. is kinda no no..  that will create forward port conflicts :innocent: